geode-management/build.gradle

```diff
@@ -49,7 +49,9 @@ dependencies {
   testCompileOnly(platform(project(':boms:geode-all-bom')))
   testCompileOnly('io.swagger:swagger-annotations')
 
-  integrationTestImplementation(project(':geode-dunit'))
+  integrationTestImplementation(project(':geode-dunit')) {
```

Comment:
geode-dunit needs geode-core, which needs geode-management. What is the actual concern in the testing here?